java/create-terrain-from-a-local-raster/README.md

```diff
@@ -0,0 +1,57 @@
+# Create terrain from a local raster
+
+Set the terrain surface with elevation described by a raster file.
+
+![Create terrain from a local raster](create-terrain-from-a-local-raster.gif)
```

Comment:
lets stick with .pngs for screenshots, otherwise the sample viewer may not be able to render screenshots correctly (gifs could be a good idea going forward though...)